src/saml2/utility/config.py

```diff
@@ -0,0 +1,2 @@
+class ConfigValidationError(Exception):
+    pass
```

Comment:
Why is this under the utility module and not under the config module?

Reply:
It's a leftover, I'll move it :+1: